SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/CrumbManager.swift

```diff
@@ -43,7 +43,7 @@ class CrumbStack {
             setLocationAndSize(byIndex:index, withWidth: width)
             label.textAlignment = .center
             label.textColor = UIColor.white
-            label.backgroundColor = [UIColor.darkGray, UIColor.lightGray][index % 2]
+            label.backgroundColor = self.cardData.color.getDarker().getDarker()
```

Comment:
I noticed that the function `getDarker` is always being invoked twice in succession. Was wondering why you choose to do it this way since the brightness parameter of `getDarker` function could be varied.

Reply:
You're right haha. I struggled with it myself. Usually getDarker (which is a common color function in other languages/frameworks) darkens the image by a significant but not so substantial amount. I thought we might use this function elsewhere to get a darker color somewhere between the original card color (fairly light) and the breadcrumb color (fairly dark), which is why I decided to allow myself to call it twice here even though it's the only usage.

Reply:
👍